.github/scripts/auto_review.py

```
@@ -109,11 +188,21 @@ def main():
             all_changes_text = "\n".join(all_changes)
-            comment = generate_gpt_comment(all_changes_text)
-            post_pr_comment(REPO, PR_NUMBER, comment, GITHUB_TOKEN)
+            pr_title = os.getenv("PR_TITLE", "")
+            changed_files_str = ", ".join(changed_filenames)
+            
+            # 한 번의 API 호출로 description과 review 모두 생성
+            description, review = generate_gpt_comment(all_changes_text, pr_title, changed_files_str)
+            
+            # PR description 업데이트
+            update_pr_description(REPO, PR_NUMBER, description, GITHUB_TOKEN)
+            print(f"[SUCCESS] PR 본문이 성공적으로 업데이트되었습니다.")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)